a11y_pygments/a11y_dark/README.md

```diff
@@ -12,13 +14,20 @@ Highlight color: ![#ffd9002e](https://via.placeholder.com/20/ffd9002e/ffd9002e.p
 
 **WCAG compliance**
 
-| Color                                                        | Hex       | Ratio     | Normal text | Large text |
-| ------------------------------------------------------------ | --------- | --------- | ----------- | ---------- |
-| ![#d4d0ab](https://via.placeholder.com/20/d4d0ab/d4d0ab.png) | `#d4d0ab` | 9.04 : 1  | AAA         | AAA        |
-| ![#ffa07a](https://via.placeholder.com/20/ffa07a/ffa07a.png) | `#ffa07a` | 7.12 : 1  | AAA         | AAA        |
-| ![#f5ab35](https://via.placeholder.com/20/f5ab35/f5ab35.png) | `#f5ab35` | 7.25 : 1  | AAA         | AAA        |
-| ![#ffd700](https://via.placeholder.com/20/ffd700/ffd700.png) | `#ffd700` | 10.09 : 1 | AAA         | AAA        |
-| ![#abe338](https://via.placeholder.com/20/abe338/abe338.png) | `#abe338` | 9.29 : 1  | AAA         | AAA        |
-| ![#00e0e0](https://via.placeholder.com/20/00e0e0/00e0e0.png) | `#00e0e0` | 8.59 : 1  | AAA         | AAA        |
-| ![#dcc6e0](https://via.placeholder.com/20/dcc6e0/dcc6e0.png) | `#dcc6e0` | 8.9 : 1   | AAA         | AAA        |
-| ![#f8f8f2](https://via.placeholder.com/20/f8f8f2/f8f8f2.png) | `#f8f8f2` | 13.28 : 1 | AAA         | AAA        |
+| Color                                                        | Hex       | Ratio   | Normal text | Large text |
+| ------------------------------------------------------------ | --------- | ------- | ----------- | ---------- |
+| ![#d4d0ab](https://via.placeholder.com/20/d4d0ab/d4d0ab.png) | `#d4d0ab` | 9.0 : 1 | AAA         | AAA        |
+
```

Comment:
Those blank lines seem to break the table in rendered view.

Reply:
I think they are inserted by a missing `-` in `{% endfor %}` in jinja, and we should be able to generate them properly.